########## buttons ##############
bool x
bool square
bool triangle
bool circle

bool PS
bool share
bool options

bool L1
bool R1
bool L3
bool R3

###################################



############# Arrows ###############

bool UpArrow
bool DownArrow
bool LeftArrow
bool RightArrow

################################



########## Analog ###############
Analog motion
##############################